spec/system/replies/new_spec.rb

```diff
@@ -391,19 +399,24 @@
         fill_in id: "reply_content", with: reply.content
         click_button "Add More Replies"
       end
-      expect(page).to have_no_selector('.error')
+
+      # TODO: add positive selector check
```

Comment:
(are these TODOs for a follow-up PR?)

Reply:
...no, I missed those when i was going back through, whoops